### PR TITLE
Create functionality

### PR DESCRIPTION
**Pull request by alfonsomartinezs**

### CODE REVIEW

File: src/mockRocks.js [hunk 1]
```diff
@@ -27,6 +27,13 @@ const rocks = [
     brilliance: 0,
     hardness: 6,
   },
+  {
```
Comment (sjproctor) [inline src/mockRocks.js:30]: Yeeessss!

File: src/pages/RockNew.js [hunk 2]
```diff
@@ -1,10 +1,85 @@
...
+import {Redirect} from 'react-router-dom'
 
 class RockNew extends Component {
+  constructor(props){
+    super(props)
+    this.state ={
+      form: {
+        name:"",
+        color:"",
+        brilliance:'NaN',
```
Comment (sjproctor) [inline src/pages/RockNew.js:12]: I am curious about the values for brilliance and hardness. You have default values that are a string of not a number?

**Review by sjproctor — APPROVED** [review]
Looks good!

File: src/pages/RockNew.js (hunk 2 above)
Comment (charper95) [reply src/pages/RockNew.js:12]: We thought about it, and we will just use empty strings.

**Pull request merged**